src/lib.rs

```diff
@@ -30,6 +30,9 @@ pub mod daemon;
 /// Error handling.
 pub mod errors;
 
+/// Interfaces for journald logging.
+pub mod journal;
```

Comment:
I think we can call this `logging` or similar. There is a counterpart for reading journald log, which we are not going to cover here. It's better to make our intention explicit.